src/components/themed/Fade.js

```diff
@@ -27,6 +27,7 @@ const FadeComponent = ({ visible: propsVisible, hidden, children }: Props) => {
     })
   }
 
+  // eslint-disable-next-line
```

Comment:
These should be `eslint-disable-next-line rule-name`, so we know what rule is being switched off. Also, these cleanups should be in a separate commit.